docs/conf.py

```diff
@@ -358,7 +356,8 @@ def setup(app):
 
 # Intersphinx configuration
 intersphinx_mapping = {
-    'aiida': ('http://aiida-core.readthedocs.io/en/latest/', None),
+    'aiida':
+    ('http://aiida-core.readthedocs.io/en/fix-4200-read-the-docs/', None),
```

Comment:
note this needs to be changed back to `latest` before merging, once https://github.com/aiidateam/aiida-core/pull/4203 is merged